### PR TITLE
Add cookie banner and enable gtag

### LINKED ISSUE #932: Enable page count analytics + cookie banner

### PR DESCRIPTION
**Pull request by e111077**
Fixes #932

We should first enable page count analtics

Links to test:

light bg:
https://pr922-8f2ca78---lit-dev-5ftespv5na-uc.a.run.app/

dark bg:
https://pr922-8f2ca78---lit-dev-5ftespv5na-uc.a.run.app/playground
https://pr922-8f2ca78---lit-dev-5ftespv5na-uc.a.run.app/tutorials/intro-to-lit

Followed semantics and DOM structure of the angular and firebase ones

### CODE REVIEW

**Comment by e111077** [comment]
oops, screenshot tests. I mean what's so bad about this test result?

**Comment by aomarks** [comment]
I think it looks a bit weird on the playground and tutorials pages, because the background color is the same as the one on that page. What if it was black?

**Comment by aomarks** [comment]
> I think it looks a bit weird on the playground and tutorials pages, because the background color is the same as the one on that page. What if it was black?

A shadow could also help. The one at https://web.dev/ is black with shadow.

**Comment by e111077** [comment]
OMG TESTS PASS! I've also added a screenshot test for the banner

**Comment by aomarks** [comment]
Curious where did the language for this come from? The web.dev one is "We serve cookies on this site to analyze traffic, remember your preferences, and optimize your experience."

**Comment by aomarks** [comment]
> What if it was black?

How did black look?

**Comment by e111077** [comment]
oh oops here's the black one

image

**Comment by augustjk** [comment]
> Curious where did the language for this come from?

Yes. Do we need to say cookies from Google?

File: packages/lit-dev-content/src/components/litdev-cookie-banner.ts [hunk 1]
```diff
@@ -1,9 +1,15 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
```
Comment (aomarks) [inline packages/lit-dev-content/src/components/litdev-cookie-banner.ts:3]: 2022

**Comment by e111077** [comment]
We finally got the go-ahead from launch review

File: packages/lit-dev-content/src/components/litdev-cookie-banner.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      font-weight: bold;
+    }
+  `;
+  override render() {
+    return html`
+      <div>
+        <h2 class="visually-hidden">Cookies consent notice</h2>
+        <div class="message">
+          Lit.dev uses cookies from Google to deliver and enhance the quality of
+          ts services and to analyze traffic.
```
Comment (aomarks) [inline packages/lit-dev-content/src/components/litdev-cookie-banner.ts:70]:
```suggestion
          its services and to analyze traffic.
```

Comment (e111077) [reply packages/lit-dev-content/src/components/litdev-cookie-banner.ts:70]: TYPESCRIPT SERVICES

**Review by aomarks — APPROVED** [review]
Screenshot needs updating I assume?

**Comment by e111077** [comment]
IDK why the tests keep getting cancelled

**Comment by e111077** [comment]
test failures are from another PR that got into main. Will fix in followup

**Pull request merged**